### PR TITLE
Native put along axis

### PR DESCRIPTION
**Pull request by Rubtsowa**
tests:
test_put_along_axis_val_int
test_put_along_axis1
test_put_along_axis2

### CODE REVIEW

**Review by shssf — APPROVED** [review]

File: dpnp/dpnp_iface_indexing.py [hunk 1]
```diff
@@ -445,9 +448,31 @@ def put_along_axis(arr, indices, values, axis):
             if not all(val_list):
                 pass
             else:
-                return dpnp_put_along_axis(arr, indices, values, axis)
+                if dpnp.isscalar(values):
```
Comment (samir-nasibli) [inline dpnp/dpnp_iface_indexing.py:451]: Don't forget to mention all the limitations in docstrings, please.

**Review by samir-nasibli — COMMENTED** [review]
Do you test this with numpy original tests?
It seems you don't have also tests for corner cases.

Need some updates and more test coverage on other PR(s).
